cmd/helm/template_test.go

```diff
@@ -141,6 +141,12 @@ func TestTemplateCmd(t *testing.T) {
 			cmd:    fmt.Sprintf("template '%s'", deletevalchart),
 			golden: "output/issue-9027.txt",
 		},
+		{
+			// Same as above, but ensures this still works when '-f' is used: https://github.com/helm/helm/issues/12488
+			name:   "ensure nil/null values pass to subcharts delete values when '-f' is used",
+			cmd:    fmt.Sprintf("template '%s' -f %s/values.yaml", deletevalchart, deletevalchart),
+			golden: "output/issue-9027.txt",
```

Comment:
Why is this `golden` file the same as the test before it but the input is different? Should this have a different output and a different `golden` file?